### PR TITLE
KNCK-3775 - gomaz cloudsearch : no search result case  

### PR DESCRIPTION
**Pull request by nalin-humin**
JIRA link: https://humindevelopment.atlassian.net/browse/KNCK-3775

Reviewers: @dosheawork, @nick-at-humin

### CODE REVIEW

**Comment by dosheawork** [comment]
Needs a test.

**Comment by nalin-humin** [comment]
d61f7b2

File: cloudsearch/cloudsearch.go [hunk 1]
```diff
@@ -102,10 +102,6 @@ func (cs *csClient) Search(query Query) ([]Result, error) {
 
 	hits := search_result.Hits
 
-	if hits.Found == 0 {
```
Comment (dosheawork) [inline cloudsearch/cloudsearch.go]: Why not short circuit and return empty results with no error?

Comment (dosheawork) [inline cloudsearch/cloudsearch.go]: nm

**Comment by dosheawork** [comment]
LGTM :+1:

**Pull request merged**